emdcbackend/emdcbackend/views/Maps/MapClusterToJudge.py

```diff
@@ -64,10 +64,26 @@ def delete_cluster_judge_mapping_by_id(request, map_id):
     return Response({"detail": "Cluster To Judge Mapping deleted successfully."}, status=status.HTTP_200_OK)
 
 
+def delete_cluster_judge_mapping_by_id_nonhttp(map_id):
```

Comment:
rename this to something like delete cluster judge mapping, the nonhttp tag on the end isn't consistent with the rest of the codebase.